routers/api/v2/users.go

```diff
@@ -16,25 +16,29 @@ import (
 // Response: token string
 // Headers:  Authorization <- token
 func (r *apiV2Router) Login(ctx *gin.Context) {
-	email := ctx.PostForm("email")
-	if len(email) == 0 {
+	var req struct {
+		Email    string `form:"email"`
+		Password string `form:"password"`
+	}
+	ctx.Bind(&req)
```

Comment:
I ignored the error returned by `ctx.Bind` here and in `Register` because we're binding to strings and since `ctx.PostForm` already simply returns a string, there's no way for us to get an error here.